bin/player-board.rb

```diff
@@ -0,0 +1,22 @@
+class Player
+  attr_accessor :name, :order
+  @@track = 0
+  def initialize()
+    @@track += 1
+    @name = "Player #{@@track}"
+    @order = @@track
+  end
+end
+
+class Board
+  attr_accessor :grid
```

Comment:
I suggest just using `attr_reader` here. We don't want outside objects to directly manipulate the value of @grid